modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/resource/v1_0/MessageBoardMessageResourceImpl.java

```diff
@@ -79,7 +79,6 @@
 import java.io.Serializable;
 
 import java.util.Collections;
-import java.util.HashMap;
```

Comment:
Uhhhh this should throw a SF in master, right?

Reply:
It was changed by Brian in the last PR that I sent. Correctly because I left a variable typed as HashMap instead of as Map, but he forgot to remove the include. That is the reason it was like that in master

Reply:
Yep, I saw the commit :)